xml/System.Reflection.Metadata.Ecma335/MethodBodyStreamEncoder.xml

```diff
@@ -75,12 +75,12 @@
         <Parameter Name="attributes" Type="System.Reflection.Metadata.Ecma335.MethodBodyAttributes" />
       </Parameters>
       <Docs>
-        <param name="instructionEncoder">To be added.</param>
-        <param name="maxStack">To be added.</param>
-        <param name="localVariablesSignature">To be added.</param>
-        <param name="attributes">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="instructionEncoder">The instruction encoder.</param>
+        <param name="maxStack">The max stack.</param>
+        <param name="localVariablesSignature">The local variables signature handle.</param>
+        <param name="attributes">The method body attributes.</param>
+        <summary>Encodes a method body and adds it to the method body stream.</summary>
```

Comment:
Each of the overloads has the same description; they should be dismbiguated.

Reply:
Done. Can you review them, please?